CICD/github/launch_load_tests.sh

```
@@ -18,10 +18,10 @@ export NVM_DIR="$HOME/.nvm"
 source ./bin/load_environment_light.sh;
 cd bin
 ./startOpfab.sh prod
-cd ../src/test/gatling
+cd ../test/gatling
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)